apps/E2E/src/Menu/pages/MenuPageObject.ts

```diff
@@ -35,32 +32,45 @@ class MenuPageObject extends BasePage {
    */
   async openMenu(): Promise<void> {
     if (!(await this.menuIsExpanded())) {
-      await this.clickItem(MenuComponentSelector.MenuTrigger);
+      await this.click(this._menuTrigger);
       await this.waitForMenuToOpen();
     }
   }
 
   async closeMenu(): Promise<void> {
     if (await this.menuIsExpanded()) {
-      await this.sendKey(MenuComponentSelector.FirstMenuItem, Keys.ESCAPE);
-      await this.waitForCondition(async () => (await this.menuIsExpanded()) === false, 'The Menu did not close.', this.waitForUiEvent, 500);
+      await this.sendKeys(this.getMenuItem(MenuItem.First), [Keys.ESCAPE]);
+      await this.waitForCondition(
```

Comment:
nit could we create a `waitForMenuToClose()` function to mimic the `openMenu()` and `waitForMenuToOpen()` functions? I could see us adding a test to close the menu by clicking somewhere else (instead of pressing 'ESC') in the future and having `waitForMenuToClose()` as a separate function would help with that. I also think it makes the code easier to read since there would be an `openMenu/closeMenu` function and `waitForMenuToOpen/waitFoMenuToClose` function instead of just a `closeMenu` function but maybe that's just me.